### PR TITLE
Foxy devel

### PR DESCRIPTION
**Pull request by padiln**
The changes made provide better support for ROS2.  This includes updating the launch files to use the new python standard and changing from the catkin build to ament.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by mm318** [comment]
Thank you very much for your contribution!

It looks like this pull request will conflict with https://github.com/aws-robotics/aws-robomaker-small-warehouse-world/pull/4. I propose merging the relevant changes from that pull request into this one.

**Comment by padiln** [comment]
@mm318 
I have merged all of @joe28965 working into this PR, and updated the documentation.

**Comment by joe28965** [comment]
Seems like we've done some nice work, the two of us. I will close #4 for it is no longer relevant.

This PR will also cover the issue raised in #3 When this PR is merged, that issue can be closed.

File: package.xml [hunk 1]
```diff
@@ -8,11 +8,14 @@
...
-  <buildtool_depend>catkin</buildtool_depend>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+  
   <exec_depend>gazebo_ros</exec_depend>
   <exec_depend>gazebo</exec_depend>
   <exec_depend>gazebo_plugins</exec_depend>
+
   <export>
-    <gazebo_ros plugin_path="${prefix}/lib" gazebo_media_path="${prefix}" gazebo_model_path="${prefix}/models"/>
```
Comment (ojasjoshi) [inline package.xml:16]: I don't see envhooks for plugin_path and gazebo_media_path. In that case, these paths should still be exported from here?

**Comment by ojasjoshi** [comment]
Looks good to me. Thanks for the contribution!

**Pull request merged**